bootstrap/sql/mysql/create_tables.sql

```diff
@@ -352,3 +352,31 @@ CREATE TABLE IF NOT EXISTS component (
   timestamp BIGINT,
   PRIMARY KEY (id)
 );
+
+CREATE TABLE IF NOT EXISTS topology_test_run_case (
+  id BIGINT AUTO_INCREMENT NOT NULL,
+  topologyId BIGINT NOT NULL,
+  versionId BIGINT NOT NULL,
+  records TEXT NOT NULL,
+  timestamp BIGINT,
+  PRIMARY KEY (id),
+  FOREIGN KEY (topologyId) REFERENCES topology(id),
+  FOREIGN KEY (versionId) REFERENCES topology_version(id)
+);
+
+CREATE TABLE IF NOT EXISTS topology_test_run_histories (
+  id BIGINT AUTO_INCREMENT NOT NULL,
+  topologyId BIGINT NOT NULL,
+  testCaseId BIGINT NOT NULL,
+  finished CHAR(5) NOT NULL,
+  success CHAR(5) NOT NULL,
+  expectedOutputRecords TEXT,
+  actualOutputRecords TEXT,
+  matched CHAR(5),
```

Comment:
Do we need both success and matched columns? I did a quick search and saw setMatched being called. But setSuccess was not called anywhere.